build-packages/magento-scripts/lib/tasks/magento/setup-magento/disable-page-builder.js

```diff
@@ -0,0 +1,11 @@
+const magentoTask = require('../../../util/magento-task');
+
+/**
+ * @type {() => import('listr2').ListrTask<import('../../../../typings/context').ListrContext>}
+ */
+const disablePageBuilder = () => ({
+    title: 'Disabling page builder in Magento',
+    task: (ctx, task) => task.newListr(magentoTask('config:set cms/pagebuilder/enabled 0'))
```

Comment:
So now we will check in DB if pagebuilder has value and if it is > 0.
In case if it is not present there (like by default it is enabled and not present in DB), then we write in DB value to disable it.
If 0 or NaN (can be in broken DB, but it will show broken page on admin panel). In this case we do nothing.

Didn't implemented for start command, because it is kind not correct, it will always disable pagebuilder even if it is not required at all. Even if we gonna state flag for it if it is already disabled, I don't think it is necessary at all, only for when installing scandipwa theme.